modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/spaces/SpaceGeneralSettings.tsx

```diff
@@ -49,6 +49,10 @@ export default function SpaceGeneralSettings({
 	setSpace?: React.Dispatch<React.SetStateAction<any>>;
 	space: Space;
 }) {
+	const [originalERC, setOriginalERC] = useState(
```

Comment:
Can you please rename it to `initialErc`? It makes more sense to me. Also, the `externalReferenceCode` cannot be null, so you can avoid `?`

`const [initalERC, setInitialERC] = useState(space.externalReferenceCode);`